x-pack/plugins/ml/common/util/anomaly_utils.ts

```diff
@@ -219,29 +222,50 @@ export function getSeverityColor(normalizedScore: number): string {
 }
 
 /**
- * Returns a label to use for the multi-bucket impact of an anomaly
- * according to the value of the multi_bucket_impact field of a record,
- * which ranges from -5 to +5.
- * @param multiBucketImpact - Value of the multi_bucket_impact field of a record, from -5 to +5
+ * Returns whether the anomaly record should be indicated in the UI as a multi-bucket anomaly,
+ * for example in anomaly charts with a cross-shaped marker.
+ * @param anomaly Anomaly table record
  */
-export function getMultiBucketImpactLabel(multiBucketImpact: number): string {
-  if (multiBucketImpact >= MULTI_BUCKET_IMPACT.HIGH) {
-    return i18n.translate('xpack.ml.anomalyUtils.multiBucketImpact.highLabel', {
-      defaultMessage: 'high',
-    });
-  } else if (multiBucketImpact >= MULTI_BUCKET_IMPACT.MEDIUM) {
-    return i18n.translate('xpack.ml.anomalyUtils.multiBucketImpact.mediumLabel', {
-      defaultMessage: 'medium',
-    });
-  } else if (multiBucketImpact >= MULTI_BUCKET_IMPACT.LOW) {
-    return i18n.translate('xpack.ml.anomalyUtils.multiBucketImpact.lowLabel', {
-      defaultMessage: 'low',
-    });
-  } else {
-    return i18n.translate('xpack.ml.anomalyUtils.multiBucketImpact.noneLabel', {
-      defaultMessage: 'none',
-    });
+export function isMultiBucketAnomaly(anomaly: AnomalyRecordDoc): boolean {
+  if (anomaly.anomaly_score_explanation === undefined) {
+    return false;
   }
+
+  const sb = anomaly.anomaly_score_explanation.single_bucket_impact;
+  const mb = anomaly.anomaly_score_explanation.multi_bucket_impact;
+
+  if (mb === undefined || mb === 0) {
+    return false;
+  }
+
+  if (sb !== undefined && sb > mb) {
+    return false;
+  }
+
+  if ((sb === undefined || sb === 0) && mb > 0) {
+    return true;
+  }
+
+  if (sb !== undefined && mb > sb) {
+    return (((mb - sb) * mb) / sb) * 1.7 >= 2;
```

Comment:
Can we add in a comment to add some context for this formula? E.g. why are we multiplying by 1.7?